application/controllers/cron/Keywords_stats.php

```diff
@@ -0,0 +1,35 @@
+<?php if (!defined('BASEPATH')) exit('No direct script access allowed');
+
+class Keywords_stats extends CI_Controller
+{
+	public function update_keywords_stats()
+	{
+		// Some keywords are not in use at all.
+		// Initialise instances field to zero before tallying instances.
+		
+		$sql = '
+			UPDATE keywords
+			SET instances = 0';
+		$this->db->query($sql);
+		
+		$sql = '
+			UPDATE keywords
+			JOIN
+			(SELECT pk.keyword_id as keyword_id, COUNT(pk.project_id) AS count
+        		FROM project_keywords pk
+        		GROUP BY 1
+			) as sub
+			ON keywords.id = sub.keyword_id
+			SET keywords.instances = sub.count ';
```

Comment:
You're even using `count` here yourself ;)

Reply:
You're quite right.